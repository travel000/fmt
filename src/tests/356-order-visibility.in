<?php
//passes:Default,OrganizeClass
class A {
	public function d(){}
	protected function b(){}
	private function c(){}
	public function a(){}
}