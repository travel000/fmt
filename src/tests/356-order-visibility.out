<?php
//passes:Default,OrganizeClass
class A {
	public function a() {}

	public function d() {}

	protected function b() {}

	private function c() {}
}